cases/templates/cases/case_detail_staff.html

```diff
@@ -94,14 +94,72 @@ <h2 class="lbh-heading-h2 section-heading">Noise Source</h2>
 <p>Hackney Estate? {{ case.get_estate_display }}
       </p>
         <p class="text-smaller">{{ case.ward }} ward</p>
+
+        <div class="case-detail__map" id="map"><div id="leaflet"></div></div>
```

Comment:
The map is a bit small when we’re shoe-horning it into this column layout. What do you think about moving it up, immediately after the "Noise source" heading? Here it is, with `1rem` of margin below it:

![Screenshot 2021-10-01 at 14 09 26](https://user-images.githubusercontent.com/739624/135625189-1707fa25-fcc3-4085-ad8c-0f4ac16559cd.png)